python/ray/client_builder.py

```diff
@@ -134,11 +137,12 @@ def _init_args(self, **kwargs) -> "ClientBuilder":
         if kwargs.get("runtime_env") is not None:
             self.env(kwargs["runtime_env"])
             del kwargs["runtime_env"]
-        if not kwargs:
-            return self
-        unknown = ", ".join(kwargs)
-        raise RuntimeError(
-            f"Unexpected keyword argument(s) for Ray Client: {unknown}")
+        if kwargs:
+            self._remote_init_kwargs = kwargs
+            unknown = ", ".join(kwargs)
+            logger.info("Passing the following kwargs to ray.init() "
+                        f"on the server: {unknown}")
```

Comment:
@ijrsvt @richardliaw @pcmoritz , I think this is a very good message, but since ray init and client are combined in ray init, everyone using client will most likely get this info if they were passing arguments to init, do you see this as an issue?

Reply:
technically also job_config and runtime env and namespace are on the server.

Reply:
Great idea @ckw017 !!